src/emu/debugger/src/elements/imgui_debugger_install_device.cpp

```diff
@@ -81,7 +81,7 @@ namespace eka2l1 {
                 const std::string install_device_welcome_msg = common::get_localised_string(localised_strings,
                     "install_device_wizard_welcome_msg");
 
-                ImGui::TextWrapped(install_device_welcome_msg.c_str());
+                ImGui::TextWrapped("%s",install_device_welcome_msg.c_str());
```

Comment:
This is intentional for now sorry... I think you have to lower down this one level if it's an error. Currently ImGui does not detect newline from the final formatted string but only in the type format string (first argument), so we have to do this :(

Reply:
I added -Wno-format-security for that file, that seems to fix my build error as well.